src/utils.js

```diff
@@ -6,7 +6,32 @@ function remove(array, element) {
   array.splice(array.indexOf(element), 1);
 }
 
+const mockGithub = {
```

Comment:
`*-spec.js` are test files. What about making a `test-utils.js` file inside the `spec` folder instead?